### DISCUSSION TITLE
Having difficulty adding POIs

### DISCUSSION THREAD

**Discussion by Riksu9000** [streetcomplete/StreetComplete#5696] [Feedback]
Hi. I recently started using this app, and it's been fun. It's easy to add details to existing objects. I was planning to map commercial services one street at a time. This is because I think it would be easier to do all at once, rather than try to find which one's were missed later on.

The problem I'm having is finding the correct place type. For one, I feel like there's an issue with searching, where it doesn't always show results that should match a part of a word, but most of the time I don't even know what the exact type should be. In this case I would like to pick something more generic, and maybe add more detail at home. For example I think it would be valuable if I could add just the name of a place and a short description as fixme if necessary, then it would be easier to specify the types at home. For now I can only add a note.

Similarly, I'm having difficulty adding playgrounds. I can add each toy in the playground, but can't find a type to map just some playground. I don't think I'll get into that kind of micromapping yet 😄.

**Comment by Riksu9000** [comment]
The category based building type selector that also allows generic building types is awesome. I'm sure something similar for things and places would be extremely useful.

**Comment by matkoniecz** [comment]
Which specific searches you tried and they failed? Are you using English on phone or some other language?

**Reply by Riksu9000** [reply]
For example, I can't find playgrounds and I was having difficulty finding a type for an employment agency, because I didn't know what it was called. On the list of common places there is "Pikaruokaravintola" (Fast food restaurant). If I search for ravintola, this doesn't show up. It's hard to say what places are missing from search, because I don't know the full list of places.

**Reply by matkoniecz** [reply]
So you use Finnish, right?

This project uses iD tagging schema and their translations for such points as fast food eateries.

You can help translate at https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md#translating - maybe Finnish translation is incomplete?

IIRC playground areas (or playground points) are not mappable with StreetComplete.

**Reply by matkoniecz** [reply]
For ravintola... 

I would say that ravintola should be added as name or synonym of restaurant to iD presets in Finnish translation ( https://en.wiktionary.org/wiki/ravintola )

But I see it at https://github.com/openstreetmap/id-tagging-schema/blob/main/dist/translations/fi.json

Are you using SC with Finnish language? What exactly you searched? Can you add screenshot?

Were you using object overlay (green background for icon) or place overlay (orange background for layer icon)? Restaurants will be in place overlay and will NOT be in things overlay.

**Reply by Riksu9000** [reply]
I try to use finnish, but it's missing a lot of words, so I often have to search using english. I'll see if I can help translate it.

In this video I try to find Pikaruokaravintola using different words. I also found that typing english words while finnish is set is giving inconsistent results, but when english is set, it's consistent. I guess this is sort of as expected, but it's inconvenient when a lot of words aren't translated.

https://github.com/streetcomplete/StreetComplete/assets/37774658/c540d35f-0c83-4961-b984-021da8e235a8

**Reply by matkoniecz** [reply]
Yeah, improving Finnish translations would definitely help iD and StreetComplete and rapid and GoMap!! users (and likely others)

**Comment by mnalis** [comment]
I agree that contributing id-tagging-schema Finnish translations would help many other people in different OSM apps (Vespucci and Every Door also use those), so thanks for considering helping with those @Riksu9000! Improved translations should then be present automatically in next StreetComplete release.

As a workaround for POIs that you cannot find by name (or which are not currently available in StreetComplete, like `playground`), you can long-press on the map on location where they should be and choose `Create new note`. 
That will create [OSM Note](https://wiki.openstreetmap.org/wiki/Notes) with text (and optionally a pictures) which you can later add  in desktop editor of your choice (like iD or JOSM).

**Reply by matkoniecz** [reply]
> As a workaround for POIs that you cannot find by name (or which are not currently available in StreetComplete, like playground), you can long-press on the map on location where they should be and choose Create new note.

I would not really recommend this as some (notably post boxes, due to technical issues with iD presets) are not shown so it would often result in making notes about already existing ones.

**Reply by mnalis** [reply]
Interesting, you have a point there! 
Although I would assume that mapper would see that on the first run they've made, and thus stop adding new notes about postboxes (and potentially other objects which do not show) afterwards... 

I was thinking mostly about regular places like specific shops/restaurants/etc... It would be nice if that postbox-related issue was resolved, though.

**Reply by matkoniecz** [reply]
> It would be nice if that postbox-related issue was resolved, though.

If anyone is interested in writing code to fix it: it is https://github.com/westnordost/osmfeatures/issues/23

**Comment by Riksu9000** [comment]
An update on my experience. With more practice, I can often find a fitting tag for places, however I have also encountered places where there is only a name on a sign, and literally no other information about the place. Now I often take photos instead.

I have added more translations and I learned that you can add synonyms, which I think will help greatly as well.

I still think that search is sometimes behaving unexpectedly, and I remember reading that some of it is intended to reduce the overwhelming number of options, even though there's only one correct answer and it might get hidden without indication because of it, but I digress.

I also remember reading that playgrounds were explicitly omitted, which is a shame since I keep encountering them, and they would be much easier to tag on-site.

So I still have minor gripes here, but I have found my workflow of simply taking more photos, which gets me past anything I come across.
